### ISSUE TITLE
Case Sensitivity issues with "./localStorage-ttls"

### ISSUE THREAD

**Issue by boringgeek** [HenrikJoreteg/ampersand-local-cache-mixin#1]
Getting the following error on `npm start`

```
ERROR in ./~/ampersand-local-cache-mixin/ampersand-local-cache-mixin.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./localStorage-ttls in /var/www/node/ghublabler/node_modules/ampersand-local-cache-mixin
 @ ./~/ampersand-local-cache-mixin/ampersand-local-cache-mixin.js 6:14-44
```

This is due to Line 6 of ampersand-local-cache-mixin.js:

```
var storage = require('./localStorage-ttls')
```

The file is installed as "localstorage-ttls.js".  This causes a file not found error.

**Comment by HenrikJoreteg** [comment]
sorry, 'bout this Curtis. Thanks @klaemo for the fix.

**Comment by HenrikJoreteg** [comment]
published to npm as `3.0.1`

**Comment by boringgeek** [comment]
Wooho! thanks @HenrikJoreteg  @klaemo :)
